<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FMsgEventDialog</class>
 <widget class="QDialog" name="FMsgEventDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>340</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>340</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>340</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>561</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>551</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string>Event Manager</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Message:</string>
    </property>
   </widget>
   <widget class="QLabel" name="msgNameLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Super Message to go in USA with family!</string>
    </property>
   </widget>
   <widget class="QLabel" name="msgDescriptioLabel">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>371</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Description:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Description:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>531</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Play it, when the following contitions are met</string>
    </property>
    <widget class="QComboBox" name="conditionComboBox">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>INDEPENDENTLY</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>AND</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OR</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="byTimerCheckBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>By time</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="byIOEventCheckBox">
     <property name="geometry">
      <rect>
       <x>332</x>
       <y>30</y>
       <width>91</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>By IO event</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="timeGroupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>141</width>
       <height>101</height>
      </rect>
     </property>
     <property name="title">
      <string>Time</string>
     </property>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>118</width>
        <height>70</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QDateTimeEdit" name="dateTimeEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="onceRadioButton">
         <property name="text">
          <string>Once</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QRadioButton" name="repeatRadioButton">
         <property name="text">
          <string>Repeatedly</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="ioGroupBox">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>332</x>
       <y>50</y>
       <width>181</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>DI Trgigger Options</string>
     </property>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>151</width>
        <height>100</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>DI0</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="di0TrigComboBox">
         <item>
          <property name="text">
           <string>NONE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RISING EDGE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FALLING EDGE</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>DI1</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="di1TrigComboBox">
         <item>
          <property name="text">
           <string>NONE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RISING EDGE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FALLING EDGE</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>DI2</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="di2TrigComboBox">
         <item>
          <property name="text">
           <string>NONE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RISING EDGE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FALLING EDGE</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>DI3</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="di3TrigComboBox">
         <item>
          <property name="text">
           <string>NONE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>RISING EDGE</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FALLING EDGE</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FMsgEventDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FMsgEventDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
